website/layouts/partials/hooks/body-end.html

```diff
@@ -0,0 +1,34 @@
+<script>
+
+    //create the html element with the version string
+    function WarningElement(version) {
+        let RawElement = `
+        <div class="content deprecation-warning">
+            <h3>
+                You are viewing documentation for Karpenter version: ${version}
+            </h3>
+            <p> Karpenter ${version} documentation is no longer actively maintained. The version you are currently viewing is a static snapshot. For up-to-date documentation, see the 
```

Comment:
Should we say something less scary, like "Karpenter ${version} documentation is not the latest stable release. The documentation version you are currently viewing is a static snapshot. ..."

Reply:
Sure